### ISSUE TITLE
Express area covered in MoF

### ISSUE THREAD

**Issue by peterdesmet** [inbo/esas2obis#6]
See https://obis.org/manual/darwincore/#sampling. Using `sampleSizeValue` and `sampleSizeUnit`, but as a measurement or fact.

**Comment by peterdesmet** [comment]
@nicolasvanermen is it possible to calculate the density for each observation?

**Comment by nicolasvanermen** [comment]
No... Some surveys apply a very wide transect, and for such surveys in
particular distance analysis prior to density calculation is absolutely
necessary...
